app/controllers/reactions_controller.rb

```diff
@@ -42,28 +42,15 @@ def create
     authorize Reaction
     Rails.cache.delete "count_for_reactable-#{params[:reactable_type]}-#{params[:reactable_id]}"
     category = params[:category] || "like"
-    reaction = Reaction.where(
-      user_id: current_user.id,
-      reactable_id: params[:reactable_id],
-      reactable_type: params[:reactable_type],
-      category: category,
-    ).first
-    if reaction
+    if (reaction = Reaction.where(set_reaction_args).first)
       reaction.user.touch
       reaction.destroy
-      Notification.send_reaction_notification_without_delay(reaction, reaction.reactable.user)
-      Notification.send_reaction_notification_without_delay(reaction, reaction.reactable.organization) if organization_article?(reaction)
+      send_reaction_notification(reaction, without_delay: true)
```

Comment:
I'm not sure we're actually improving the code here because although we've removed one line we have to add a param to use in the called method as a switch...